float64[] q_max
float64[] theta
float64[] phi

---------

bool success
